
package uade.fain.ia.tpo.interfaces.soap;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 2.7.7.redhat-1
 * 2014-11-10T22:23:15.912-03:00
 * Generated source version: 2.7.7.redhat-1
 */

@WebFault(name = "OrdenDespachoYaExisteException", targetNamespace = "http://soap.interfaces.tpo.ia.fain.uade/")
public class OrdenDespachoYaExisteException_Exception extends java.lang.Exception {
    
    private uade.fain.ia.tpo.interfaces.soap.OrdenDespachoYaExisteException ordenDespachoYaExisteException;

    public OrdenDespachoYaExisteException_Exception() {
        super();
    }
    
    public OrdenDespachoYaExisteException_Exception(String message) {
        super(message);
    }
    
    public OrdenDespachoYaExisteException_Exception(String message, Throwable cause) {
        super(message, cause);
    }

    public OrdenDespachoYaExisteException_Exception(String message, uade.fain.ia.tpo.interfaces.soap.OrdenDespachoYaExisteException ordenDespachoYaExisteException) {
        super(message);
        this.ordenDespachoYaExisteException = ordenDespachoYaExisteException;
    }

    public OrdenDespachoYaExisteException_Exception(String message, uade.fain.ia.tpo.interfaces.soap.OrdenDespachoYaExisteException ordenDespachoYaExisteException, Throwable cause) {
        super(message, cause);
        this.ordenDespachoYaExisteException = ordenDespachoYaExisteException;
    }

    public uade.fain.ia.tpo.interfaces.soap.OrdenDespachoYaExisteException getFaultInfo() {
        return this.ordenDespachoYaExisteException;
    }
}
